platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/generate-platform-diagram.sh

```diff
@@ -0,0 +1,30 @@
+########################################################################################################################
+## Creates high level groupings of platform components.
+## Substitutes spammy wires, and collapses groupings with transformers.
```

Comment:
I suggest removing this comment, not really necessary when there is only one command in this file.

Also, please add `#!/usr/bin/env bash` as the first line in this script.